textattack/loggers/file_logger.py

```diff
@@ -14,6 +14,8 @@
 class FileLogger(Logger):
     """Logs the results of an attack to a file, or `stdout`."""
 
+    nocolor = False
```

Comment:
Can you make `nocolor` an attribute of the specific `FileLogger` instance (e.g. `self.nocolor`) instead of the entire class? Remember that if you set the `nocolor` a class-level attribute, then it's going to set `color_method` to `None` for actual text file logging too.